src/ore/src/tracing.rs

```diff
@@ -487,7 +492,11 @@ where
     #[cfg(feature = "tokio-console")]
     let stack = stack.with(tokio_console_layer);
     let stack = stack.with(sentry_layer);
-    stack.init();
+
+    let global_subscriber: Arc<dyn Subscriber + Send + Sync + 'static> = Arc::new(stack);
+    let _ = GLOBAL_SUBSCRIBER.set(global_subscriber);
+
+    Arc::clone(&GLOBAL_SUBSCRIBER.get().unwrap()).init();
```

Comment:
For this to work I think we need to ensure that all entrypoints (testing + binary) call `mz_ore::tracing::configure`. I'm not sure if this is the case at the moment.